### ISSUE TITLE
Issues with the latest release: Cannot find module 'babel/polyfill'

### ISSUE THREAD

**Issue by danielkoo** [CascadeEnergy/dynamoDb-marshaler#8]
Hi there,

I believe v1.1.0 is giving me this:

```
(master) addl1006018:tmp $ cat index.js 
var db = require('dynamoDb-marshaler');

(master) addl1006018:tmp $ cat package.json 
{
  "name": "blah",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "dynamodb-marshaler": "^1.1.0"
  }
}

(master) addl1006018:tmp $ node index.js 
module.js:338
    throw err;
          ^
Error: Cannot find module 'babel/polyfill'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/dank/Documents/code/tmp/node_modules/dynamoDb-marshaler/marshalItem.js:15:1)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
(master) addl1006018:tmp $ 
```

Forcing the version to 1.0.0, and it works again.

Please let me know if you need anything else, thanks.

Daniel

**Comment by nackjicholson** [comment]
Thanks, and sorry about that. I will fix it as soon as I'm able, and let you know.

**Comment by nackjicholson** [comment]
Thank you for the issue. Fixed with version `1.1.1`. Let me know if you have anymore problems.

**Comment by danielkoo** [comment]
Thanks for the prompt response; I can load 1.1.1 up fine now, ta!
